src/arrayMethodReduce.js

```diff
@@ -4,8 +4,19 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, ...initialValue) {
+    const prev = initialValue;
```

Comment:
Woulde be better to rename this variable to "accumulator"

Reply:
👌 fixed